syntax = "proto3";

import "proto/vector.proto";
package core.pb;


message Config
{
    string window_name = 1;
    Vec2i window_size = 2;
    int32 framerate_limit = 3;
    bool fullscreen = 4;
    bool vertical_sync = 5;
    bool enable_debug = 6;
    int32 major_version = 7;
    int32 minor_version = 8;
    bool es = 9;
    bool no_imgui = 10;
}